simpledotorg/simple-server
# Add bootboot to pave the way for Rails 6

see https://github.com/Shopify/bootboot

**Story card:** [ch6351](https://app.shortcut.com/simpledotorg/story/6351/setup-dual-dependency-approach-to-prepare-for-rails-6-and-beyond)

## Because

We want to be able to easily manage major dependency updates, such as Rails and Ruby

## This addresses

Adds bootboot to make it easier to manage dual gemfiles in the app.

## Test instructions

check out branch, run `bundle`, verify the normal gem update process works as normal.

DEPENDENCIES_NEXT will actually _install_ gems from the Gemfile_next file, so once we get Rails 6 setup there that is how you would switch over to it.

```
# This will update Gemfile.lock and Gemfile_next.lock and install the gems
# specified in Gemfile.lock:
$ bundle update some_gem
# This will actually install the gems specified in Gemfile_next.lock
$ DEPENDENCIES_NEXT=1 bundle install
```
